api/server/router/volume/volume_routes.go

```diff
@@ -159,25 +159,35 @@ func (v *volumeRouter) deleteVolumes(ctx context.Context, w http.ResponseWriter,
 	}
 	force := httputils.BoolValue(r, "force")
 
-	version := httputils.VersionFromContext(ctx)
-
+	// First we try deleting local volume. The volume may not be found as a
+	// local volume, but could be a cluster volume, so we ignore "not found"
+	// errors at this stage. Note that no "not found" error is produced if
+	// "force" is enabled.
 	err := v.backend.Remove(ctx, vars["name"], opts.WithPurgeOnError(force))
```

Comment:
I'm also looking if we can remove the `force` from the backends, and instead handle "not-exist" errors on the caller site, but looks like that may need more work ("purge" does more than only ignoring "not-exist" errors, as it also updates state in the backend to remove volumes from the store if the (_remote_ ?) volume store (no longer) has a volume.